### DISCUSSION TITLE
Problem with Knowledge base plugin after deploy to production

### DISCUSSION THREAD

**Discussion by ejjaquet** [GuavaCZ/filament-knowledge-base#57] [General]
Hi!

I am working on a Filament application with the Knowledgebase plugin. Recently I moved my code to a production server but then I get an error as soon as I load the KB panel:

Undefined constant Filament\View\PanelsRenderHook::PAGE_SUB_NAVIGATION_SELECT_BEFORE

I don't use the RenderHook myself in the code, and locally it is working fine. Does anybody recognize this?

Kind regards,

Erik-Jan

**Comment by lukas-frey** [comment]
Hi, knowledge base doesn't use that render hook either.
